ci-operator/step-registry/openshift/e2e/azure/cgroupsv2/openshift-e2e-azure-cgroupsv2-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: openshift-e2e-azure-cgroupsv2
```

Comment:
Is this used anywhere?  I don't see it mentioned elsewhere in this PR

Reply:
Thanks for pointing that out, I will remove these files.